shotover/src/codec/cassandra.rs

```diff
@@ -7,18 +7,21 @@ use anyhow::{anyhow, Result};
 use atomic_enum::atomic_enum;
 use bytes::{Buf, BufMut, BytesMut};
 use cassandra_protocol::compression::Compression;
+use cassandra_protocol::crc::{crc24, crc32};
```

Comment:
since we have this now, it would be good to go through and ensure we always use the imported version in this file instead of fully specifying it.